src/huggingface_hub/repository.py

```diff
@@ -86,6 +86,17 @@ def __repr__(self):
         return f"[{self.title} command, status code: {status}, {'in progress.' if not self.is_done else 'finished.'} PID: {self._process.pid}]"
 
 
+def run_subprocess(command, folder, check=True) -> subprocess.CompletedProcess:
+    return subprocess.run(
+        command.split(),
```

Comment:
This is wrong I think, you really want to pass list of arguments and not split arbitrarily.

For instance on `filename` containing spaces it would break:

`filename = "my filename.txt`,  command is now: `["xxx", "my", "filename.txt"], instead of expected ["xxx", "my filename.txt"].

Reply:
yes!

Reply:
Thanks for the review, will adapt the draft